### PR TITLE
[release-1.23] announce 1.21.6

### PR DESCRIPTION
**Pull request by jaellio**
Security release related to https://github.com/istio/istio.io/pull/15710

### CODE REVIEW

File: content/en/news/releases/1.21.x/announcing-1.21.6/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+This release fixes the security vulnerabilities described in our September 19th post, [ISTIO-SECURITY-2024-006](/news/security/istio-security-2024-006).
+This release note describes what’s different between Istio 1.21.5 and 1.21.6.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** `PILOT_SIDECAR_USE_REMOTE_ADDRESS` functionality on sidecars to support setting internal addresses to mesh network rather than localhost to prevent header sanitization if `envoy.reloadable_features.explicit_internal_address_config` is enabled.
+
+- **Fixed** VirtualMachine WorkloadEntry locality label missing during auto registration.
```
Comment (dhawton) [inline content/en/news/releases/1.21.x/announcing-1.21.6/index.md:19]:
```suggestion
- **Fixed** `VirtualMachine` `WorkloadEntry` locality label missing during auto registration.
```

**Review by dhawton — COMMENTED** [review]
minor nit, otherwise LGTM

**Review by dhawton — APPROVED** [review]
LGTM if it merges tonight, otherwise let's bump publish date

**Comment by wilsonwu** [comment]
Also need a manual cherry pick to master.

**Comment by dhawton** [comment]
> Also need a manual cherry pick to master.

Yup.. I will be taking care of it once they are merged and the security releases are all handled.

**Review by dhawton — APPROVED** [review]

**Pull request merged**